### DISCUSSION TITLE
Unable to copy backend service

### DISCUSSION THREAD

**Discussion by shreebadiger** [learndevopsonline/learndevopsonline#1686] [Q&A]
![error](https://github.com/learndevopsonline/learndevopsonline/assets/145904607/12ff2b4a-b64b-4dd3-a040-bfe6ae56964c)

**Comment by shreebadiger** [comment]
Workstation ip 54.175.139.34
Connected this server 10.10.4.251 through workstation

**Reply by sreedharm07** [reply]
Y builtin.user ?? 
It should be copy or template  
Change it
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/41d024fb-b3d4-440b-98a9-947b168a92ae)

**Comment by shreebadiger** [comment]
![Screenshotx](https://github.com/learndevopsonline/learndevopsonline/assets/145904607/2392f6ce-070a-45ab-b963-4b1ad6ae0ca0)

**Comment by shreebadiger** [comment]
Is it right ??

**Comment by shreebadiger** [comment]
Is ther any answer??
